src/input.rs

```diff
@@ -0,0 +1,78 @@
+use sfml::window::Key;
+use std::sync::mpsc::{Receiver, Sender};
+
+pub struct Input {
+    current_input: char,
+    game_off: bool,
+    game_off_rx: Receiver<bool>,
+}
+
+impl Input {
+    pub fn new(game_off_rx: Receiver<bool>) -> Input {
+        Input {
+            current_input: ' ',
```

Comment:
Dlaczego nie przekazujesz `current_input` od razu enuma `Action`? To by było dużo bardziej przejrzyste niż te "tajemnicze" znaki i nie potrzebna byłaby konwersja z `char`  do `Action`.